src/loaders/raw/tvgRawLoader.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All rights reserved.
+
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+#ifndef _TVG_RAW_LOADER_H_
+#define _TVG_RAW_LOADER_H_
+
+class RawLoader : public Loader
+{
+public:
+    const uint32_t* content = nullptr;
+    bool copy;
+
+    RawLoader();
+    ~RawLoader();
+
+    using Loader::open;
+    bool open(const uint32_t* data, uint32_t w, uint32_t h, bool copy) override;
+    using Loader::read;
+    bool read() override;
+    bool close() override;
+
+    using Loader::scene;
```

Comment:
question: what is  using Loader::scene;  for ?

Reply:
Parent class(Loader), declares scene() as pure virtual.
When a child class implements this, if it is unnecessary, convert pure virtual to virtual.
So child classes must be defined using using.

Reply:
please double check it, otherwise remove them.

Reply:
I was misunderstanding. I deleted the code